docs/dev/secret-contracts.md

```diff
@@ -92,9 +92,9 @@ The `src` folder contains the following files:
 
 ##### `contract.rs` 
 
-This file contains functions which define the available contract operations. The functions which all secret contracts contain will be: `init`, `handle`, and `query`. 
+This file contains functions which define the available contract operations. The functions which all Secret Contracts contain will be: `init`, `handle`, and `query`. 
 
-- `init` is called once at instantiation of the secret contract with 3 parameters: `deps`, `env`, and `msg`. The internal state (`State` struct imported from `state.rs`)of the secret contract is initialized with these parameters.
+- `init` is called once at instantiation of the Secret Contract with 3 parameters: `deps`, `env`, and `msg`. These parameters are used to initialize the internal state (the `State` struct imported from `state.rs`) of the Smart Contract.
```

Comment:
These parameters initialize the internal state (the `State` struct imported from `state.rs`) of the Secret Contract.

Reply:
I think it's safe to just refer to smart contracts as "Secret Contracts". Still working out what should be canonical here.